app/players/elves/elf.py

```diff
@@ -0,0 +1,17 @@
+from players.player import Player
+
+
+class Elf(Player):

```

Comment:
Your `Elf` class should also inherit from `ABC` because it does not implement all abstract methods from the `Player` class.